sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -276,6 +276,7 @@
         public static final String CURRENT_VERSION = "2018-12-31";
         public static final String QUERY_VERSION = "1.0";
         public static final String AZURE_COSMOS_PROPERTIES_FILE_NAME = "azure-cosmos.properties";
+        public static final String AZURE_COSMOS_SPARK_31_PROPERTIES_FILE_NAME = "azure-cosmos-spark-3-1_2-12.properties";
```

Comment:
`azure-cosmos-spark-3-1_2-12.properties` file doesn't exist.

we should add a similar file. please see here:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/resources/azure-cosmos.properties

Reply:
actually I think this may not work. see the structure of the file: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/resources/azure-cosmos.properties

this file relies on the current project maven coordinates. whereas for spark it is a different project.